docs/docs/general/02-Getting Started/annotate-interactively.md

```diff
@@ -1,10 +1,17 @@
+---
+sidebar_position: 6
+---
+
+
 # Annotate Interactively
 
 Similar to [annotate image](annotate-image.md), the askui client offers an interactive annotation tool which helps with the test creation and debugging of failed tests.
 
 ## How It Works
 
-The interactive annotation command requests the askui server to take a screenshot of the specified screen. Then, an AI model is used to annotate the image. After that, a full-screen window appears. Inside this window, boundary boxes enclosing the UI elements detected are going to appear. You can hover over the boxes to see the boxes' specifications, e.g., text detected, element type etc.
+The interactive annotation command `annotateInteractively()` requests the askui server to take a screenshot of the specified screen. Then, an AI model is used to annotate the image. After that, a full-screen window appears. Inside this window, boundary boxes enclosing the UI elements detected are going to appear. You can hover over the boxes to see the boxes' specifications, e.g., text detected, element type etc.
+
+📌 *Press `esc` to exit the full-screen window.*
 
 ![Interactive Annotation](./interactive-annotate.gif)
```

Comment:
Please move the gif to `static/img/gif` to save repository space.

Otherwise, they would be copied with every version 😉